### ISSUE TITLE
empty 'div' on pages with no box

### ISSUE THREAD

**Issue by bennie99** [psi-4ward/boxes4ward#2]
It would be nice on pages that are excluded from this Modul to make the empty 'div' disappear.

**Comment by christophbach** [comment]
You can simply add an if loop around the box div checking wether there’s content or not.

``` php
<?php if ($this->content) { ?>
  <!-- original/your template -->
<?php } ?>
```

**Comment by psi-4ward** [comment]
implemented in 771bdc11a82e121842d1b399c07b9d099dde1ea7
